tests/src/core/testqgsgdalprovider.cpp

```diff
@@ -86,6 +87,11 @@ void TestQgsGdalProvider::initTestCase()
 
   mSupportsNetCDF = GDALGetDriverByName( "netcdf" ) != NULL;
 
+  // Disable creation of .aux.xml (stats) files during test run,
+  // to avoid modifying .zip files.
+  // See https://github.com/qgis/QGIS/issues/48846
+  setenv( "GDAL_PAM_ENABLED", "NO", 1 );
```

Comment:
setenv() is not portable:
```suggestion
  CPLSetConfigOption( "GDAL_PAM_ENABLED", "NO" );
```

Reply:
Thanks ! Interesting the same setting is already found in other tests..